javacutil/build.gradle

```diff
@@ -12,6 +12,17 @@ dependencies {
     // External dependencies:
     // If you add an external dependency, you must shadow its packages.
     // See the comment in ../build.gradle in the shadowJar block.
+
+    /// TODO: The below is temporary until the `removeDuplicates` routine is moved into plume-lib.
```

Comment:
I see that you also added `removeDuplicates` to SystemUtil.  Is the dependency on Guava still required?  Although, Guava is shadowed in `checker.jar`, it's not shadowed in the dataflow and javacutil artifacts we publish.  If we add a dependency to Guava here, it could cause a classpath conflict for users who uses those artifacts.

Reply:
`removeDuplicates` depends on Guava.  I'll move the method into plume-util later this month, and then this dependency won't be needed any more.

Reply:
So at that point javacutil will have a transitive dependency on Guava.  Are you going to shadow Guava in plume-util?

Reply:
OK, I have removed the dependency on Guava.